orte/tools/orterun/orterun.c

```diff
@@ -2266,20 +2265,8 @@ static int process(char *orig_line, char *basename, opal_cmd_line_t *cmd_line,
            conditions. */
         tmp = opal_argv_join(argv, ' ');
 
-        /* We do not support launching a debugger that requires the
-           -np value if the user did not specify -np on the command
-           line. */
-        if (used_num_procs && 0 == num_procs) {
-            free(tmp);
-            tmp = opal_argv_join(orterun_argv, ' ');
-            orte_show_help("help-orterun.txt", "debugger requires -np",
-                           true, (*new_argv)[0], argv[0], tmp,
-                           (*new_argv)[0]);
-            /* Fall through to free / fail, below */
-        } 
-
         /* Some debuggers do not support launching MPMD */
-        else if (single_app && NULL != strchr(tmp, ':')) {
```

Comment:
@rhc54 This looked like totally dead code (and therefore also enabled me to remove a block from help-orterun.txt).

Can you confirm that it's ok to remove this block?